sponsorship/src/main/java/com/kyriba/conference/sponsorship/api/PlanController.java

```diff
@@ -0,0 +1,71 @@
+package com.kyriba.conference.sponsorship.api;
+
+import com.kyriba.conference.sponsorship.api.dto.PlanCancellationRequest;
+import com.kyriba.conference.sponsorship.api.dto.PlanRegistrationRequest;
+import com.kyriba.conference.sponsorship.service.PlanService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * @author M-ASL
+ * @since v1.0
+ */
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@RequestMapping(value = "/api/v1/sponsorship/plan",
+    consumes = { MediaType.APPLICATION_JSON_UTF8_VALUE }, produces = { MediaType.APPLICATION_JSON_UTF8_VALUE })
+@Api(value = "Register a new sponsorship plan")
+public class PlanController
+{
+  private final PlanService planService;

```

Comment:
We don't need implementation at that stage

Reply:
I believe, it is similar to https://github.com/gariklolkin/conference/pull/10#discussion_r286013126, so for now, I'd prefer to leave it as is